groupbalancer.go

```diff
@@ -93,31 +98,28 @@ func (r rangeStrategy) AssignGroups(members []memberGroupMetadata, topicPartitio
 // 		C1: [1, 4]
 // 		C2: [2, 5]
 //
-type roundrobinStrategy struct{}
+type RoundRobinGroupBalancer struct{}
```

Comment:
@achille-roussel I was thinking about maybe leaving this type unexported and having a constant instead:
```suggestion
var RoundRobin GroupBalancer = roundRobinGroupBalancer{}
```

Does that make for a clearer API for the client, or do you think it's better to have the type exported?